### ISSUE TITLE
Wayland support

### ISSUE THREAD

**Issue by morrolinux** [apirrone/Memento#84]
Hi,
Thanks for this awesome piece of software!
Now that Microsoft has also done something similar (Recall) the hype on the subject has definitely been lit and I think you should probably expect a surge in visits to this repo very soon if not already. 

That said, most modern distributions (and users) already use Wayland by default and it would be cool for Memento to support Wayland as well. `pyscreenshot` claims wayland support, so perhaps that would be a good place to start.

**Comment by apirrone** [comment]
You're right, it goes in my todo for when I have time to put some work into memento :) 

Thanks !
